netket/graph/lattice.py

```diff
@@ -246,6 +315,114 @@ def atoms_coord(self):
         """
         return self._atoms_coord
 
+    @property
+    def coords(self):
+        """
+        Returns list of coordinates of lattice points
+        """
+
+        return self._coords
+
+    def translation_perm(self):
```

Comment:
Docstring missing and type annotation missing. 
Can you write what this method does?

Regardless, i think this is an internal method? If so it should be prepended with an underscore.